cmd/snap/cmd_sign.go

```diff
@@ -115,22 +118,13 @@ func (x *cmdSign) Execute(args []string) error {
 		return err
 	}
 
-	if x.Chain != "" {
-		var known assertKnower
-
-		switch x.Chain {
-		case "remote":
-			known = knownRemoteWithFallback
-		case "direct":
+	// --direct implies --chain
```

Comment:
not sure we want to do this? but not strong opinion but if we do we should document it above

Reply:
@pedronis 

I did this for 2 reasons:

1) it *sort of* mirrors the `--direct implies --remote` behavior in `snap known`

2) it was the easy way to avoid defining wha happens when `--direct` is specified without `--chain`

I am happy to change, but then need guidance on what should happen if  someone calls `snap sign --direct` without `--chain`. Silently ignore `--direct`? Warn and continue? Fail?